cli/jira_config_gen/__init__.py

```diff
@@ -39,16 +39,33 @@
     default="/tmp/jira.config",
     type=click.STRING,
 )
+@click.option(
+    "--template_dir",
+    help="Directory holding templates",
+    required=True,
+    default="/firewatch/cli/templates",
+    type=click.STRING,
+)
+@click.option(
+    "--template_filename",
```

Comment:
WDYT about get only template file name path and then you can get the base directory from it?
```
from pathlib import Path
template_directory = Path(template_filename).parent
```